docs/modules/publisher-directmethods.md

```diff
@@ -0,0 +1,325 @@
+[Home](../../readme.md)
+
+### Configuration via IoT Hub Direct methods ###
+
+OPC Publisher implements the following [IoT Hub Direct Methods](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-direct-methods) which can be called from an application (anywhere in the world) leveraging the [IoT Hub Device SDK](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-sdks).
+
+The following direct methods are inherited from 2.5.x in addition to some new ones:
+
+- PublishNodes_V1
+- UnpublishNodes_V1
+- UnpublishAllNodes_V1
+- GetConfiguredEndpoints_V1
+- GetConfiguredNodesOnEndpoints_V1
+- GetDiagnosticInfo_V1
+- AddOrUpdateEndpoints_V1
+
+To understand the terminologies and migration path from version 2.5.x of OPC Publisher to 2.8.2s, please refer to [this](publisher-dmmigration.md) migration document.
+
+
+
+Now let's dive into each direct method request and response payloads with examples.
+
+#### PublishNodes_V1**
+
+PublishNodes enables a client to add a set of nodes to be published for a specific DataSetWriter. When a DataSetWriter already exists, the nodes shall be incrementally added to the very same dataset. When it does not already exist, a new DataSetWriter shall be created with the initial set of nodes contained in the request.
+
+  *Request:* follows strictly the request payload schema as described, the OpcNodes attribute being mandatory.
```

Comment:
Where is the described schema ? Let's add description of the JSON payload schema above before talking about specific direct methods similar to original document.